shavar/views/__init__.py

```diff
@@ -191,7 +194,11 @@ def format_downloads(request, resp_payload):
                 baseurl = sblist.settings.get('redirect_url_base')
                 if not baseurl:
                     baseurl = _setting(request, 'shavar', 'redirect_url_base')
-                fudge = os.path.join(baseurl, lname, "%d" % chunk.number)
+                if list_ver is not None:
+                    fudge = os.path.join(
+                        baseurl, lname, list_ver, "%d" % chunk.number)
```

Comment:
@groovecoder, it seems like we will not need this after all. Which means we probably don't need https://github.com/mozilla-services/shavar-list-creation/pull/99 since the check to see the latest list file uses the path `/tracking/<list name>/<version>` and retrieval to download the said file simply returns the chunknum file name in the `/<file name>`. Should we remove it or should we keep it to be consistent with the way the `/tracking` is organized?

Reply:
Whatever changes we don't need we can drop. Would like to change shavar as little as possible to make this work to minimize risk. And we'll get rid of shavar when we move to remote settings anyway.